quarkchain/evm/state.py

```diff
@@ -703,6 +704,8 @@ def ephemeral_clone(self):
         s.journal = copy.copy(self.journal)
         s.cache = {}
         s.qkc_config = self.qkc_config
+        if timestamp:
```

Comment:
how about simply do 

```
s.timestamp = self.timestamp
```

Reply:
will remove the update of timestamp here since the timestamp has already been updated when creating the evm state while this is a copy of that state.